### ISSUE TITLE
Default value for property of "integer enum" type caused POJO compilation error

### ISSUE THREAD

**Issue by s13o** [joelittlejohn/jsonschema2pojo#706]
POJO like this generated wrong and can't be compiled

```json
 {
    "javaType" : "com.example.enums.IntegerEnumToSerialize",
    "properties" : {
       "testEnum" : { 
          "type" : "integer",
          "enum" : [1, 2, 3],
          "javaEnumNames" : ["One", "Two", "Three"],
          "default": 3
      }
    }
}
```

It can be reproduced easily with jsonschema2pojo/jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/EnumIT.java test

**Comment by joelittlejohn** [comment]
Not the error here relates to how the default value is passed to the enum fromValue method - it's currently always passed as a string e.g. (from the above schema)

```java
@JsonProperty("testEnum")
private IntegerEnumToSerialize.TestEnum testEnum
   = IntegerEnumToSerialize.TestEnum.fromValue("3");
...
@JsonCreator
public static IntegerEnumToSerialize.TestEnum fromValue(Integer value) {
...
```

**Comment by s13o** [comment]
I know, fixed already.
Will try to commit soon (again :) )

**Comment by joelittlejohn** [comment]
Closed by #706 from @thachhoang. Thanks for raising this and helping to improve the project @s130!
